packages/bpk-component-calendar/src/Week.js

```diff
@@ -21,9 +21,6 @@ import React, { Component } from 'react';
 import { cssModules, deprecated } from 'bpk-react-utils';
 
 import {
-  getDay,
-  getFirstDayOfWeekend,
```

Comment:
Does this need to be removed from bpk-react-utils, or is it used elsewhere ?

Reply:
Ooo good question will take a look!